app/models/ability.rb

```diff
@@ -0,0 +1,18 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize user
```

Comment:
`user ` này lấy từ đâu ra em ơi

Reply:
@VuThiTranVan Dạ user được truyền vào sẽ là giá trị trả về của hàm current_user (hàm built-in của devise hoặc mình có thể tự định nghĩa). Nếu người dùng đã đăng nhập thì hàm current_user sẽ trả về object của người dùng đó, hoặc trả về nil nếu chưa đăng nhập ạ.